backend/tracim_backend/app_models/applications.py

```diff
@@ -1,80 +1,59 @@
 # coding=utf-8
 import typing
 
-from tracim_backend.app_models.contents import ContentType
-from tracim_backend.models.roles import WorkspaceRoles
-
 if typing.TYPE_CHECKING:
-    from tracim_backend.config import CFG  # noqa: F401
-    from tracim_backend.app_models.contents import ContentStatus  # noqa: F401
+    from tracim_backend.config import CFG
+    from tracim_backend.lib.utils.app import TracimApplication
+    from tracim_backend.lib.utils.app import TracimContentType  # noqa F401
 
 
-class Application(object):
+class TracimApplicationInContext(object):
```

Comment:
We do need app in context for color (color of app need cfg to be obtained.).

Reply:
probably better to name this class something like `ConfigReadyApplication` or something like this? What do you think?

Reply:
I'm not sure that a good naming. "InContext" class (this is not the first one, we do have them for database classe too) are object use to get extend information about an object. It's needed for REST api but we do not use them to act, it's just for showing information extended by the current context.